M/MPIFiles/Compat.toml

```diff
@@ -15,3 +15,6 @@ julia = "0.7-1"
 
 ["0.5.3-0"]
 julia = ["0.7", "1"]
+
+["0.6.2-0"]
+HDF5 = "0.0.0-0.12.1"
```

Comment:
Would it be better to fix HDF5? If a user already has the latest HDF5 version Pkg will just choose an old version of this package that is compatible with HDF5.

Reply:
The problem is not HDF5 but that they implemented a feature, that breaks my package. I will try fixing that, but its not trivial. Until that happens, I wanted to make a release, that make clear that version > 0.12.1 is currently not working.